### PR TITLE
capability to delete worklogs previously uploaded

### PR DESCRIPTION
**Pull request by scrungus**

### CODE REVIEW

File: jira-log.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ set -e
 #worklog -i SCRUM-42 -t 15 -c "Daily Scrum" 
 
 #####configuration
-jiraUser="mark@stackhpc.com"
+jiraUser="tyler@stackhpc.com"
```
Comment (markgoddard) [inline jira-log.sh]: :(

Comment (markgoddard) [reply jira-log.sh]: Maybe it should be mandatory.

File: jira-delete.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash 
+set -eu 
+
+#####configuration
+jiraUser="tyler@stackhpc.com"
```
Comment (markgoddard) [inline jira-delete.sh]: Could you make this a mandatory argument? It gives me a 404 if I try to delete a work log with your email.

File: jira-delete.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+jiraDomain="https://stackhpc.atlassian.net"
+#####configuration
+
+if [[ -f ~/.jira-user ]]; then
+        jiraUser=$(cat ~/.jira-user)
+else
+    if [ -z "$jiraUser" ]; then
+    	read -p "Enter Username: " jiraUser
+    fi
+fi
```
Comment (scrungus) [inline jira-delete.sh:14]: @markgoddard done something similar to the api token here

Comment (markgoddard) [reply jira-delete.sh:14]: Thanks

**Review by markgoddard — APPROVED** [review]

**Pull request merged**